config/notifiers.go

```diff
@@ -392,3 +403,54 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+type JiraConfig struct {
+	NotifierConfig `yaml:",inline" json:",inline"`
+
+	Domain   string
+	User     string
+	Password Secret
+
+	Project     string                 `yaml:"project" json:"project"`
+	IssueType   string                 `yaml:"issueType" json:"issueType"`
+	Priority    string                 `yaml:"priority" json:"priority"`
+	Summary     string                 `yaml:"summary" json:"summary"`
+	Description string                 `yaml:"description" json:"description"`
+	Fields      map[string]interface{} `yaml:"fields" json:"fields"`
+
+	// Catches all undefined fields and must be empty after parsing.
+	XXX map[string]interface{} `yaml:",inline" json:"-"`
+}
+
+// UnmarshalYAML implements the yaml.Unmarshaler interface.
+func (c *JiraConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	*c = DefaultJiraConfig
+	type plain JiraConfig
+	if err := unmarshal((*plain)(c)); err != nil {
+		return err
+	}
+	if c.VSendResolved {
+		return fmt.Errorf("sending resolved notifications to Jira is not supported")
```

Comment:
a) there's no way of keeping state like the ticket ID associated with an alert. the relationship between firing and resolved alert is via the groupkey. I would have to associate the ticket with the group key somehow (e.g. a label), which is more complex than I'd like to add right now
b) I don't really want tickets to be auto-closed. if it fired, it requires investigation.